pipelines/DarkEnergyCamera/ApVerifyWithFakes.yaml

```diff
@@ -24,7 +24,7 @@ tasks:
     config:
       photoCal.match.referenceSelection.magLimit.fluxField: i_flux
       photoCal.match.referenceSelection.magLimit.maximum: 22.0
-  singleFrameWithFakes:
+  processVisitFakes:
```

Comment:
Would it be possible to delete this section so it pulls that calibrate config in from elsewhere? It's the default in `$AP_PIPE_DIR/pipelines/DarkEnergyCamera/ProcessCcd.yaml`, so maybe it is already properly imported as the default here, too?

Reply:
I could (should?) move it to an `$AP_PIPE_DIR/pipelines/DarkEnergyCamera/ApPipeWithFakes.yaml`, but it would still be asymmetric with HSC. But I don't understand what you mean by it being the default; `ProcessCcd.yaml` doesn't know about `processVisitFakes`.

Reply:
OK, scratch that, I forgot that `processVisitFakes` is code for "insert fakes into calexps and redo characterize and calibrate for reasons." Therefore, any camera-specific ProcessCcd configs *do* need to explicitly be here.

Reply:
Also scratch what I said, we specifically _don't_ want to import `ApPipeWithFakes` because of conflicts with multiple overrides to the same task.

However, @mrawls recommended on Slack that the hardcoded `isr` and `calibrate` overrides here be replaced with an import of instrument-specific `ProcessCcd`; this won't conflict with base `ApVerifyWithFakes` because the latter doesn't touch those tasks at all.